docs/data-access/data-download.md

```diff
@@ -80,31 +140,187 @@ metadata.url=https://song.rdpc.cancercollaboratory.org
 storage.url=https://score.rdpc.cancercollaboratory.org
 ```
 
-Once you have configured your `application.properties`, you will be ready to initiate your download. Run the score-client from within the `/bin` directory using the `download` command.
+Once you have configured your `application.properties`, you will be ready to use the score-client.
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+
+## Score-client Usage  
+
+This section provides information on how to use the score-client once it has been properly downloaded and configured according to the [distribution type](/docs/data-access/data-download#distributions).
+
+The score-client has the general syntax:
 
 ```shell
-> score-client-3.1.1/bin/score-client download --manifest ./directory-path/score-manifest.20200520.tsv --output-dir ./output-directory-path
+score-client [options] [command] [command options]
```

Comment:
@rosibaj Do you want to give a complete overview of the score-client usage as following and later pinpoint the three commands (download, view and mount) for accessing/downloading data. E.g.,
```
Usage: score-client [options] [command] [command options]
  Options:
        --silent
       Do not produce any informational messages
       Default: false
        --help
       Show help information
       Default: false
        --profile
       Define environment profile used to resolve configuration properties
       Default: default
        --quiet
       Reduce output for non-interactive usage
       Default: false
        --version
       Show version information
       Default: false
  Commands:
    view      Locally store/display some or all of a remote SAM/BAM file object
    version   Display application version information
    mount     Mount a read-only FUSE file system view of the remote storage repository
    url       Resolve the URL of a specified remote file object
    help      Display help information for a specified command name
    info      Display application configuration information
    manifest  Resolve a file object manifest and display it
    download  Retrieve file object(s) from the remote storage repository
    upload    Upload file object(s) to the remote storage repository
```